.swiftformat

```diff
@@ -30,7 +30,8 @@
     linebreaks, \
     modifierOrder, \
     numberFormatting, \
-    opaqueGenericParameters
+    opaqueGenericParameters, \
+    preferForLoop
```

Comment:
not particularly fan of this rule, in many cases it makes sense to use ForEach() over a for-in loop.